pkg/client/liquidation_recovery.go

```diff
@@ -0,0 +1,184 @@
+package client
+
+import (
+	"context"
+
+	"github.com/keep-network/keep-core/pkg/net"
+	"github.com/keep-network/keep-core/pkg/operator"
+	"github.com/keep-network/keep-ecdsa/pkg/chain"
+	"github.com/keep-network/keep-ecdsa/pkg/chain/bitcoin"
+	"github.com/keep-network/keep-ecdsa/pkg/ecdsa/tss"
+	"github.com/keep-network/keep-ecdsa/pkg/extensions/tbtc"
+	"github.com/keep-network/keep-ecdsa/pkg/extensions/tbtc/recovery"
+	"github.com/keep-network/keep-ecdsa/pkg/node"
+	"github.com/keep-network/keep-ecdsa/pkg/registry"
+)
+
+// TODO: Should this function be moved to `node` package under tss.Node?
+func handleLiquidationRecovery(
+	ctx context.Context,
+	hostChain chain.Handle,
+	tbtcHandle chain.TBTCHandle,
+	bitcoinHandle bitcoin.Handle,
+	networkProvider net.Provider,
+	tbtcConfig *tbtc.Config,
+	tssNode *node.Node,
+	operatorPublicKey *operator.PublicKey,
+	keep chain.BondedECDSAKeepHandle,
+	keepsRegistry *registry.Keeps,
+	derivationIndexStorage *recovery.DerivationIndexStorage,
+) error {
+	logger.Infof(
+		"starting liquidation recovery protocol for keep [%s]",
+		keep.ID(),
+	)
+
+	members, err := keep.GetMembers()
+	if err != nil {
+		logger.Errorf(
+			"failed to retrieve members from keep [%s]: [%v]",
+			keep.ID(),
+			err,
+		)
+		return err
+	}
+
+	memberID := tss.MemberIDFromPublicKey(operatorPublicKey)
+
+	memberIDs, err := tssNode.AnnounceSignerPresence(
+		ctx,
+		operatorPublicKey,
+		keep.ID(),
+		members,
+	)
+	if err != nil {
+		logger.Errorf(
+			"failed to announce signer presence on keep [%s] termination: [%v]",
+			keep.ID(),
+			err,
+		)
+		return err
+	}
+
+	chainParams, err := tbtcConfig.Bitcoin.ChainParams()
+	if err != nil {
+		logger.Errorf(
+			"failed to parse the configured net params: [%v]",
+			err,
+		)
+		return err
+	}
+
+	beneficiaryAddress, err := recovery.ResolveAddress(
+		tbtcConfig.Bitcoin.BeneficiaryAddress,
+		derivationIndexStorage,
+		chainParams,
+		bitcoinHandle,
+	)
+	if err != nil {
+		logger.Errorf(
+			"failed to resolve a btc address for keep: [%s] address: [%s] err: [%v]",
+			keep.ID(),
+			tbtcConfig.Bitcoin.BeneficiaryAddress,
+			err,
+		)
+		return err
+	}
+
+	vbyteFee, vbyteFeeError := bitcoinHandle.VbyteFeeFor25Blocks()
+	if vbyteFeeError != nil {
+		logger.Errorf(
+			"failed to retrieve a vbyte fee estimate, [%v]",
+			vbyteFeeError,
+		)
+		// Since the electrs connection is optional, we don't return the error
+	}
+	if vbyteFee == 0 {
+		vbyteFee = tbtcConfig.Bitcoin.MaxFeePerVByte
+	}
+	if vbyteFee == 0 {
+		vbyteFee = 75
```

Comment:
What do you think about pulling this 75 out into a constant and then writing a comment to explain the double `if vbyteFee == 0` check?

Reply:
Yep, makes perfect sense! The two ifs confused me when I got back to this code after a while.